lib/ingestor-api/runtime/src/validators.py

```diff
@@ -24,14 +24,14 @@ def s3_object_is_accessible(bucket: str, key: str):
     """
     Ensure we can send HEAD requests to S3 objects.
     """
-    from .config import settings
 
-    client = boto3.client("s3", **get_s3_credentials())
+    #  client = boto3.client("s3", **get_s3_credentials())
+    client = boto3.client("s3")
     try:
         client.head_object(
             Bucket=bucket,
             Key=key,
-            **{"RequestPayer": "requester"} if settings.requester_pays else {},
+            **{"RequestPayer": "requester"},
```

Comment:
Why this change?  My inclination is to keep the service configurable, it's possible that others would _not_ want to assume the data access costs.

If we do want to remove the configurable nature, then we should probably drop the `requester_pays` setting from our configuration tooling.

Reply:
As we discussed in Slack DM, this was failing in real world tests.  I'd flag this also needs further investigation.  Based on my initial research there is no cost/downside to specifying `requester-pays` for requests against buckets with no requester pays configuration.